modules/oe_showcase_news/config/install/core.entity_view_display.node.oe_news_simple.search_result.yml

```
@@ -0,0 +1,58 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - core.entity_view_mode.node.search_result
+    - field.field.node.oe_news_simple.field_news_content
+    - field.field.node.oe_news_simple.field_news_image
+    - field.field.node.oe_news_simple.field_news_publication_date
+    - image.style.medium
+    - node.type.oe_news_simple
+  module:
+    - datetime
+    - image
+    - text
+    - user
+id: node.oe_news_simple.search_result
+targetEntityType: node
+bundle: oe_news_simple
+mode: search_result
+content:
+  field_news_content:
+    type: text_summary_or_trimmed
+    weight: 4
+    region: content
+    label: above
+    settings:
+      trim_length: 350
+    third_party_settings: {  }
+  field_news_image:
+    weight: 0
+    label: above
+    settings:
+      image_style: medium
+      image_link: ''
+    third_party_settings: {  }
+    type: image
+    region: content
+  field_news_publication_date:
+    weight: 3
+    label: above
+    settings:
+      timezone_override: Europe/Brussels
+      format_type: news_full_time_format
```

```suggestion
      format_type: html_date
```
use this one till we have design